docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -501,7 +501,7 @@ Requirements for collections to be included in the Ansible Package
 To be included in the `ansible` package, collections must meet the following criteria:
 
 * :ref:`Development conventions <module_dev_conventions>`.
-* `Collection requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ (this document).
+* `Collection requirements <https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html>`_ (this document).
```

Comment:
Here we could also use a Sphinx reference, but I prefer pointing to the `devel` docs (which this link always does). So :+1:.